test/metabase/test/data/impl/get_or_create.clj

```diff
@@ -147,29 +147,65 @@
   (data-perms/set-database-permission! (perms-group/all-users) new-db-id :perms/create-queries :query-builder-and-native)
   (data-perms/set-database-permission! (perms-group/all-users) new-db-id :perms/download-results :one-million-rows))
 
+(defonce ^:private driver->loaded-datasets
```

Comment:
I think as a follow-on I'll try to implement new driver method to check if a test dataset is already loaded or not locally, and also call it automatically if the test data was created by a previous session